web/src/containers/Wallets/History.tsx

```diff
@@ -166,7 +166,7 @@ export class WalletTable extends React.Component<Props> {
             rejected: <FailIcon />,
             processing: this.props.intl.formatMessage({ id: 'page.body.wallets.table.pending' }),
             prepared: this.props.intl.formatMessage({ id: 'page.body.wallets.table.pending' }),
-            submitted: (confirmations !== undefined && minConfirmations !== undefined) ? (
+            submitted: (confirmations !== 'N/A' && confirmations !== undefined && minConfirmations !== undefined) ? (
```

Comment:
minConfirmations && confirmations && confirmations !== 'N/A'

Reply:
Thanks. I fixed it as suggested.